packages/dashboard/src/components/confirmModal/confirmModal.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { ReactElement } from 'react';
+import { Box, Button, Modal, SpaceBetween } from '@cloudscape-design/components';
+import { colorBackgroundHomeHeader } from '@cloudscape-design/design-tokens';
+
+import './confirmModal.css';
+
+interface ConfirmModalProps {
+  headerTitle: string;
+  cancelTitle?: string;
+  submitTitle: string;
+  description: ReactElement | string;
+  visible: boolean;
+  handleDismiss: () => void;
+  handleCancel: () => void;
+  handleSubmit: () => void;
+}
+
+export const ConfirmModal = ({
+  headerTitle,
+  cancelTitle = 'Cancel',
+  submitTitle,
+  description,
+  visible,
+  handleDismiss,
+  handleCancel,
+  handleSubmit,
+}: ConfirmModalProps) => {
+  return (
+    <Modal
+      visible={visible}
+      onDismiss={handleDismiss}
+      header={headerTitle}
+      data-testid='confirm-modal'
+      footer={
+        <Box float='right'>
+          <SpaceBetween direction='horizontal' size='xs'>
+            {cancelTitle && (
+              <Button variant='link' onClick={handleCancel}>
+                {cancelTitle}
+              </Button>
+            )}
+            <Button variant='primary' className='btn-custom-primary' onClick={handleSubmit} data-testid='submit'>
```

Comment:
if using Cloudscape components className is a deprecated prop, please use buttonVariants to distinguish between them

Reply:
but the cloudscape button variants do not have the required color(#ff9900 orange). so to match the required background color and border color I used className.